src/Voron/Data/Tables/Table.cs

```diff
@@ -509,8 +509,12 @@ public void Delete(long id)
             if (largeValue)
                 return;
 
-            var density = ActiveDataSmallSection.Free(id);
-            if (ActiveDataSmallSection.Contains(id) || density > 0.5)
+            var rawDataSection = ActiveDataSmallSection.Free(id);
+            if (ActiveDataSmallSection.Contains(id))
```

Comment:
shouldn't this be then `rawDataSection` instead of `ActiveDataSmallSection` ?

Reply:
It might be that it's freed from another section and not from the `ActiveDataSmallSection`.
https://github.com/ravendb/ravendb/blob/a8da3a2cfaf46a6b9e7e5d6a2a4d9d277ab5bee5/src/Voron/Data/RawData/RawDataSection.cs#L320

Reply:
yes, this is why you should use rawDataSection.Contains, right?

Reply:
Um, why this change?
You are returning the raw data section of the value, but why?
So you'll get the density, which you already previously got?

Reply:
It was done, to avoid calculation of Density, we are now only doing that if `Contains(id) == false`

Reply:
Oh, it is that expensive?

Reply:
![image](https://user-images.githubusercontent.com/6964706/205505221-b692bb6f-0cad-40d5-98c7-7cfff9008f96.png)

Reply:
@ppekrol is I use `rawDataSection` instead of `ActiveDataSmallSection` the condition will always be true.